main.go

```diff
@@ -93,6 +125,10 @@ func connectToSlice(config util.Config) SliceClients {
 	return clients
 }
 
+func init() {
+	log.SetFlags(log.LstdFlags | log.Lshortfile)
```

Comment:
Good change, could you put it in its own commit with an explanation?

Reply:
Done